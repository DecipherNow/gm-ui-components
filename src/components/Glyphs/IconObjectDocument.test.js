import React from "react";
import { shallow } from "enzyme";

import IconObjectDocument from "./IconObjectDocument.svg";

describe("IconObjectDocument", () => {
  it("matches snapshot", () => {
    const aIconObjectDocument = shallow(<IconObjectDocument />);
    expect(aIconObjectDocument).toMatchSnapshot();
  });
});