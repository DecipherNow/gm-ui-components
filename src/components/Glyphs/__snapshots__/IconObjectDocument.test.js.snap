// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconObjectDocument matches snapshot 1`] = `
<Icon
  backgroundColor="currentColor"
  backgroundOpacity={1}
  backgroundSizeRatio={1}
  borderColor="currentColor"
  borderOpacity={1}
  borderWidth={1}
  glyph=""
  glyphColor="currentColor"
  glyphName="ObjectDocument"
  glyphSizeRatio={1}
>
  <path
    d="M5.5,5 L18.5,5 C18.7761424,5 19,5.22385763 19,5.5 L19,18.5 C19,18.7761424 18.7761424,19 18.5,19 L5.5,19 C5.22385763,19 5,18.7761424 5,18.5 L5,5.5 C5,5.22385763 5.22385763,5 5.5,5 Z M8.75404165,13 C8.33982808,13 8.00404164,13.3357864 8.00404164,13.75 C8.00404164,14.1642136 8.33982808,14.5 8.75404165,14.5 L15.2540417,14.5 C15.6682552,14.5 16.0040417,14.1642136 16.0040417,13.75 C16.0040417,13.3357864 15.6682552,13 15.2540417,13 L8.75404165,13 Z M8.75404165,15.5 C8.33982808,15.5 8.00404164,15.8357864 8.00404164,16.25 C8.00404164,16.6642136 8.33982808,17 8.75404165,17 L15.2540417,17 C15.6682552,17 16.0040417,16.6642136 16.0040417,16.25 C16.0040417,15.8357864 15.6682552,15.5 15.2540417,15.5 L8.75404165,15.5 Z M8.75404165,10.5 C8.33982808,10.5 8.00404164,10.8357864 8.00404164,11.25 C8.00404164,11.6642136 8.33982808,12 8.75404165,12 L15.2540417,12 C15.6682552,12 16.0040417,11.6642136 16.0040417,11.25 C16.0040417,10.8357864 15.6682552,10.5 15.2540417,10.5 L8.75404165,10.5 Z M8.75295856,6.99999998 C8.33934316,6.99999998 8.00404164,7.3353015 8.00404164,7.74891689 C8.00404164,8.16253228 8.33934316,8.49783382 8.75295856,8.49783382 L10.7510831,8.49783382 C11.1646985,8.49783382 11.5,8.16253228 11.5,7.74891689 C11.5,7.3353015 11.1646985,6.99999998 10.7510831,6.99999998 L8.75295856,6.99999998 Z"
    fillRule="nonzero"
    id="Object-Document"
  />
</Icon>
`;
